react-native-in-app-payments-quickstart/app/components/CardsOnFileModal.tsx

```diff
@@ -46,21 +45,20 @@ export default function CardsOnFileModal({
           ? (
             <Text style={styles.noCardsText}>
               {'No cards on file found.'
-              + 'Please tap the button to add a card that you can use for future transactions.'}
+                + 'Please tap the button to add a card that you can use for future transactions.'}
             </Text>
-          ) : cardsOnFile.map(cardOnFile => (
+          ) : cardsOnFile.map((cardOnFile: any) => (
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.